README.md

```diff
@@ -86,6 +86,40 @@ public class MyConfig implements WebMvcConfigurer {
 }
 ```
 
+If you need to use additional converters you should declare `SpecificationArgumentResolver` with `ConversionService` containing required converters. 
```

Comment:
I think that conversions could be described in a separate section later in the document and it could be only mentioned here that it is possible. The idea for this readme is to show the basics quickly and not overwhelm the user with all the details.